provisioning/miniprov/hare_mp/main.py

```diff
@@ -234,6 +234,9 @@ def _start_consul(utils: Utils,
     peers = []
     for endpoint in consul_endpoints:
         key = endpoint.split('/')
+        # Considering tcp endpoints only. Ignoring all other endpoints.
```

Comment:
we should not assume that 0th position will always have tcp protocol. Code should be written in a way that should search for tcp url and use it. Tomorrow we may end up adding 3rd entry here and that should not cause a change in code

Reply:
@kupranay, we are iterating over the endpoints here and I don't see hardcoding to a certain position.

Reply:
If you are referring to `key[0]` that is about 1st part of endpoint after splitting the entry. Ofcourse, that may fail if the format of the endpoint changes, but is that expected, if so then we need to finalize the format now.

Reply:
Oh... Got it.. i missed the for loop mentioned above. This logic looks good.
One minor suggestion... should we split baed on "/" or ":" and search for tcp only instead of "tcp:"

By the way i have no concern with split with "/"

Reply:
We are using `/` as a delimiter to extract the endpoint, as we use that but the protocol we don't use, its just required to identify the correct endpoint.